src/components/Markings/DrawingClassifier.js

```diff
@@ -163,7 +190,7 @@ class DrawingClassifier extends Component {
                     isQuestionVisible={isQuestionVisible}
                     setQuestionVisibility={this.setQuestionVisibility}
                 >
-                    {isQuestionVisible ? classification : tutorial}
+                    {isQuestionVisible ? (this.props.workflowOutOfSubjects ? outOfClassifications : classification) : tutorial}
```

Comment:
How is this calculated (`workflowOutOfSubjects`)? Are we still allowing users to continue classifying when all subjects are retired? That's the current functionality of PFE.

Reply:
This should probably be taken out